pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -31,7 +31,13 @@ buildDotnetModule rec {
     hash = "sha256-FzQphMhiC1g+6qmk/R1v4rq2ldy35NcaWm0RR1UlwLA=";
   };
 
-  projectFile = "NexusMods.App.sln";
+  # If the whole solution is published, there seems to be a race condition where
+  # it will sometimes publish the wrong version of a dependent assembly, for
+  # example: Microsoft.Extensions.Hosting.dll 6.0.0 instead of 8.0.0.
+  # https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/7.0/solution-level-output-no-longer-valid
+  # TODO: do something about this in buildDotnetModule
+  projectFile = "src/NexusMods.App/NexusMods.App.csproj";
```

Comment:
@l0b0 @MattSturgeon this reduces the output size significantly (~600->100MiB).  The tests still pass, but I'm wondering if there's anything we should be concerned about losing here.

Reply:
Thanks for the ping! We found in #318647 that we needed to use the csproj to avoid other issues too.

I'm not very knowledgeable on dotnet, but I think the solutions are generated _from_ the project files anyway; so this should be fine.